betting-game-react/src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Redirect, Route, RouteProps, Switch } from 'react-router-dom';
+
+import routes, { MATCHES } from '../../config/routes';
+import NavbarButton from '../NavbarButton/NavbarButton';
+import NavbarLoginButton from '../NavbarLoginButton/NavbarLoginButton';
+import NavbarLogo from '../NavbarLogo/NavbarLogo';
+import { NavbarButtonProps, navbarButtons } from './NavbarButtonProps';
+
+const Navbar: React.FC = () => {
+  return (
+    <span>
+      <nav
+        className='flex items-center justify-between flex-wrap border-b-2 border-blue-300 bg-blue-900 p-2'
+        data-testid='Navbar'
+      >
+        <NavbarLogo />
+        <div className='block flex-grow lg:flex lg:items-center lg:w-auto'>
+          <div className='text-sm lg:flex-grow inline'>
+            {navbarButtons.map(
+              (navbarButton: NavbarButtonProps, index: number) => {
+                return (
```

Comment:
the return is unecessary, you can just leave out the curly bracket and immediately after the arrow write the component.
Also I would put line 21 in line 22 and end with the arrow